### DISCUSSION TITLE
Export Contours as Dicom RTStructs

### DISCUSSION THREAD

**Discussion by HolmlundWilliam** [HeroImaging/Community#8] [Q&A] [answered]
When I try to export contours for an image with oblique orientations I get the following error:

![image](https://user-images.githubusercontent.com/101408521/235631121-a39a2f1f-60a2-4681-89f6-8e8c99353d72.png)

Any ideas why this is happening?

**Comment by PatrikBrynolfsson** [comment, marked as answer]
Hey William!

Thanks for bringing this to our attention. This is a bug that has now been fixed, and it will be included in the next release. I will contact you regarding some testing for the next beta release if you are interested.
